index.bs

```diff
@@ -2073,7 +2073,7 @@ The following abstract operations operate on {{ReadableStream}} instances at a h
  1. Perform ! [$InitializeReadableStream$](|stream|).
  1. Let |controller| be a [=new=] {{ReadableByteStreamController}}.
  1. Perform ? [$SetUpReadableByteStreamController$](|stream|, |controller|, |startAlgorithm|,
-    |pullAlgorithm|, |cancelAlgorithm|, 0, undefined).
```

Comment:
The value 0 matches what the reference implementation does, and the reference implementation passes the tests.

Reply:
Thanks. I did not realize that [ReadableStreamDefaultControllerShouldCallPull](https://streams.spec.whatwg.org/#readable-stream-default-controller-should-call-pull) [ReadableByteStreamControllerShouldCallPull](https://streams.spec.whatwg.org/#readable-byte-stream-controller-should-call-pull) return early to so that when strategyHWM = 0, then no queuing is used. That is a very nice feature.